shinken/objects/contact.py

```diff
@@ -297,4 +297,5 @@ def explode(self, contactgroups, notificationways):
                 if not hasattr(c, 'notificationways'):
                     c.notificationways = [nw_name]
                 else:
+                    c.notificationways = list(c.notificationways)
```

Comment:
Well caught ! The problem comes from `c.notificationways` being a reference on the template property, shared across objects. When values are added in the second object, they are in fact added into the root (shared) list instance.

But I think this problem is more general and should be fixed directly in https://github.com/naparuba/shinken/blob/master/shinken/objects/item.py#L325.